<!DOCTYPE html>
<html lang="en">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <title> Vikram Gaikwad</title>

  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.0/umd/popper.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.0/js/bootstrap.min.js"></script>


  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
  <link href="https://fonts.googleapis.com/icon?family=Sora" rel="stylesheet">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
  <link href="css/blue_light.css" type="text/css" rel="stylesheet" media="screen,projection"/>
</head>

    <nav class="navbar collapse navbar-expand-lg fixed-top py-2">
      <a class="navbar-brand font-weight-bold" href="index.html"><img src="images/logo_blue.png" alt="Sangram Kadam" height="50"></a>
<button class="navbar-toggler" id="NavButton" type="button" data-toggle="collapse" data-target="#collapsibleNavbar">
    <span class="navbar-toggler-icon"></span>
  </button>

  
        <div class="collapse navbar-collapse" id="collapsibleNavbar">
            <ul class="navbar-nav ml-auto">
                <li class="nav-item active font-weight-bold">
                    <a class="nav-link" href="index.html">Home</a>
                </li>
                <li class="nav-item font-weight-bold">
                    <a class="nav-link" href="#">Research</a>
                </li>
                <li class="nav-item font-weight-bold">
                    <a class="nav-link" href="beyond_science.html"> Other Stuff</a>
                </li>
            </ul>
        </div>
    </nav>

<body>

       
        <section id="section1">
         <h3>
       Enhance sampling for protein folding
        <h3/>
        <div class="container-fluid">
            <div class="row">
                <div class="col-lg-7 col-sm-10">

                <p class="big">
                  Enhanced sampling techniques not only improve the efficiency of simulations but also enable the identification of key intermediates and transition states in the protein folding process. By capturing these elusive states, researchers gain valuable insights into the underlying mechanisms governing protein folding pathways. This enhanced understanding is critical for designing therapeutic strategies and developing interventions for protein misfolding diseases.
                <p/>

                <p class="big">
                To address this, we utilized publicly available nucleosome-resolution contact probability (Micro-C) data to systematically coarse-grain chromatin and predict important quantities for polymer representation of chromatin. We computed size distributions of chromatin beads for different coarse-graining scales, quantified fluctuations and distributions of bond lengths between neighboring regions, and derived effective spring constant values.
                <p/>

                <p class="big">
                Our findings suggest that contrary to prevalent notions, coarse-grained chromatin beads should be considered soft particles that can overlap. We derived an effective inter-bead soft potential and quantified an overlap parameter, which provides crucial insights into the nature of chromatin packing. We also computed angle distributions between neighboring bonds, which gave insights into intrinsic folding and local bendability of chromatin.
                <p/>

                <p class="big">
                Interestingly, we found that the bead sizes, bond lengths, and bond angles showed different mean behavior at the boundaries of chromatin domains. We integrated our findings into a coarse-grained polymer model and provided quantitative estimates of all model parameters, which can serve as a foundational basis for all future coarse-grained chromatin simulations.
                <p/>

                </div>
                <div class="col-lg-5 col-sm-8" style="text-align:center;">
                <img src="images/cg_chromatin_model.png" class="rounded float-left" alt="sangram" width="100%" style="margin-bottom: 40px;">
                <br>
                <p class="big"><a class="link" href="https://doi.org/10.1101/2022.08.22.504791">Click here to read more about this work</a> </p>
                </div>
            </div>
        </div>
        </section>

        <section id="section2">
         <h3>
             Disruption of polyhomeotic polymerization decreases nucleosome occupancy and alters genome accessibility
        <h3/>
        <div class="container-fluid">
            <div class="row">

                <div class="col-lg-4 col-sm-7" style="text-align:center;">
        <img src="images/prc_occupancy.jpg" class="rounded float-left" alt="PH-driven chromatin organization" width="100%" style="margin-bottom: 40px;">
                <br>
                <p class="big"><a class="link" href="http://doi.org/10.26508/lsa.202201768">Click here to read more about this work</a> </p>
                </div>


                <div class="col-lg-8 col-sm-11">
                <p class="big">
                Chromatin topology plays a crucial role in gene expression and development. Disruption of long-range chromatin contacts due to mutations in the PH protein can lead to developmental defects. Our work aimed to understand the molecular mechanism underlying this phenomenon, which could potentially provide a better understanding of the genetic basis of developmental defects and diseases. This work was in collaboration with Prof. Ajazul H Wani's group at the University of Kashmir, India.
                <p/>

                <p class="big">
                Prof. Ajazul Wani's group performed the experimental work investigating the effect of the SAM domain mutation on nucleosome occupancy and accessibility on a genome-wide scale. Our modeling work complemented the experimental work by providing insights into the interplay between distant chromatin contacts and nucleosome occupancy. We performed polymer simulations to investigate how the disruption of PH polymerization affects the organization of chromatin and nucleosome occupancy.
                <p/>

                <p class="big">
                Our polymer simulations revealed that nucleosome density increases when contacts between different regions of chromatin are established. Our results show that nucleosome density increases when contacts between different regions of chromatin are established, which suggests that  higher-order organization can have a top-down causation effect on nucleosome occupancy.
                <p/>

                <p class="big">
                In conclusion, our modeling work contributed to a better understanding of the molecular mechanism underlying the regulation of chromatin organization by studying the interplay between distant chromatin contacts and nucleosome occupancy, both of which are regulated by PH polymerization. This work could potentially provide a better understanding of the genetic basis of developmental defects and diseases.
                <p/>

                </div>
            </div>
        </div>
		</section>
		

			
		
    <div class="footer-copyright text-center py-3">Copyright Â© 2023 :
      <a href="index.html">Sangram Kadam</a> | All rights reserved.
    </div>

        		
		
		
  <!--  Scripts-->
  <script src="https://code.jquery.com/jquery-2.1.1.min.js"></script>
  <script src="js/materialize.js"></script>
  <script src="js/init.js"></script>
  <script src="js/script.js"></script>



  </body>
</html>
